codegen/codegen_test.go

```diff
@@ -857,6 +857,22 @@ func TestCodeGen(t *testing.T) {
 				llb.Dir("/etc"),
 			).Root())
 		},
+	}, {
+		"merge op",
+		[]string{"default"},
+		`
+		fs default() {
+			image "alpine"
+			merge image("root1") image("root2")
+		}
+		`, "",
+		func(ctx context.Context, t *testing.T) solver.Request {
+			return Expect(t, llb.Merge([]llb.State{
+				llb.Image("alpine"),
+				llb.Image("root1"),
+				llb.Image("root2"),
+			}))
+		},
```

Comment:
Missing test for diff op, or is this not possible?

Reply:
Added a test case for `diff`